site/content/user-guide/http-adapter.md

```diff
@@ -302,15 +293,38 @@ Response:
     HTTP/1.1 202 Accepted
     Content-Length: 0
 
-An authenticated device may also use this URI. In this case the device needs to know both the *auth-id* as well as the corresponding *device identifier*:
+### Publish Telemetry Data (authenticated gateway)
```

Comment:
Some lines in the introduction paragraph must be changed as well to reflect the now possible authentication of the gateway (instead of the device).
They only contain the device as authentication possibility, like e.g.
`The adapter verifies the credentials provided by the device against the credentials...`

Also I would now prefer if we change
`Publish Telemetry Data` to
`Publish Telemetry Data (authenticated devices)` to immediately giving the reader the information that there are different possibilities (now 3 in summary).
Same for event.

Reply:
> Some lines in the introduction paragraph must be changed as well to reflect the now possible authentication of the gateway (instead of the device).

Can you point me to the line that needs to be changed from your point of view? I can't find anything that needs to be changed AFAIC

Reply:
Yes, e.g.

The adapter verifies the credentials provided by the device against the credentials that the configured Credentials service has on record for the device.

Some rephrasing into "by the device or gateway", OR: a subsequent sentence that instead of the device a gateway acting on behalf of the device would be helpful IMHO.
As short as possible, but complete, this is what I meant.
WDYT?

Reply:
The authentication process is generic and does not really care whether it is a *device* or a *gateway* that gets authenticated. I can imagine using the abstract term *client* instead of *device*, though. Would that help?

Reply:
Yes, that would improve it.
IMHO it would be best to state at the end of the paragraph that a *client* could be a *device* or *gateway*, but I leave that up to you.
*Client* fixes the main point for me.